test/features/test_librosa_fbank.py

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
I wonder whether we should also have a test that computes the features using `LibrosaFbank` for a number of different lengths of a waveform.

Reply:
OK, The test checks that the length of the feature follows the general formula for convolution output size with +-1 frame tolerance.

Reply:
I meant a test that actually runs an object LibrosaFbank class. Just to make sure we are able to detect whenever something gets broken inside due to somebody modifying it. Right now you can see codecov complaining that the code is never run.

Reply:
I am not sure what you mean tbh. Do you want to initialize the class, but not call execute? The test can not be executed unless we install librosa in the CI. I see that the codecov is failing, but not sure what should be changed.

Reply:
I'm gonna merge for now since it is unlikely to break anything. 

I don't think I explained myself well. I meant that we should probably install librosa as part of CI and run the code that imports it (make the test conditional on its availability, see `pytest.mark.skipif`). There are tests of others feature extractor classes and `LibrosaFbank` should be included in them. But it is not urgent and we can add these later.

Reply:
If you have more time to spend on this then feel free to create a follow-up PR.